src/site/layouts/campaign_landing_page.drupal.liquid

```diff
@@ -5,28 +5,331 @@
 
 <div id="content" class="interior" data-template="node-campaign_landing_page">
   <main class="va-l-detail-page">
-    <div class="usa-grid usa-grid-full">
-      <div class="usa-width-three-fourths">
-        <!-- Draft status -->
-        {% if !entityPublished %}
-          <div class="usa-alert usa-alert-info">
-            <div class="usa-alert-body">
-              <p class="usa-alert-text">You are viewing a draft.</p>
+    <!-- Draft status -->
+    {% if !entityPublished %}
+      <div class="usa-alert usa-alert-info">
+        <div class="usa-alert-body">
+          <p class="usa-alert-text">You are viewing a draft.</p>
+        </div>
+      </div>
+    {% endif %}
+
+    <div class="campaign-landing-page">
+      <div class="hero" style="background-image: url('{{ fieldHeroImage.entity.image.url }}">
+        <!-- Hero Content -->
+        <div class="hero-content">
+          <h1>{{ title }}</h1>
+          <hr />
+          <p>{{ fieldHeroBlurb }}</p>
+
+          {% if fieldPrimaryCallToAction %}
+            <a class="usa-butto-secondary" href="{{ fieldPrimaryCallToAction.entity.fieldButtonLink.url.path }}" rel="noreferrer noopener">
```

Comment:
Did you intend to add `target="_blank"` alongside `rel="noreferrer noopener"`?

Reply:
Yupp 🙂 This is all code that will be modified, it's just boilerplate so that @ncksllvn and I can work in parallel on this next week